app/controllers/api/v1/parks_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V1::ParksController < ApplicationController
+
+  def index
```

Comment:
You need to add API controller tests for both index and show

Reply:
Added API controller tests for StatesController (see below notes re: ParksController)